x-pack/plugins/security_solution/server/lib/detection_engine/assets/security_field_map.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const securityFieldMap = {
```

Comment:
For posterity: what was the logic used to generate this list of fields?

Reply:
Answered offline: these are all the fields in the existing signals index that were NOT either:

1. part of ECS
2. identified as past/current/future CTI fields.